lesson2/random number.py

```diff
@@ -0,0 +1,8 @@
+if __name__ == '__main__':
+    import random
+    print('Input lowest ran number')
+    inp1 = int(input())
+    print('Input highest ran number')
+    inp2 = int(input())
+    num = random.randint(inp1,inp2)
```

Comment:
Add checks for errors. For example if inp1 = -2 and inp2 = -10

Reply:
Done